sousmot/sousmotapp/static/scss/main.scss

```
@@ -65,25 +65,68 @@ $navbar-item-active-color: $navbar-item-hover-color;
     }
 
     @include mobile {
-      max-width: 80vw;
+        max-width: 80vw;
     }
 }
 
 footer.footer {
     text-align: center;
-    padding:3rem 1.5rem 4rem;
+    padding: 3rem 1.5rem 4rem;
 }
 
 body {
-    display:flex;
+    display: flex;
     min-height: 100vh;
     flex-direction: column;
 }
 
 section {
-    flex:1;
+    flex: 1;
 }
 
 .has-addons-fullwidth .button.is-fullwidth {
-    width:50%;
-}
\ No newline at end of file
+    width: 50%;
+}
+
+// Game
+.table td {
+    padding: 0.5rem;
+    color: white;
+    font-size: 2rem;
+    font-weight: bold;
+    background-color: $navbar-background-color;
+
+}
+
+.table td div {
+    width: 50px;
+    height: 50px;
+    overflow: hidden;
+}
+
+.good_place {
+    background-color: $game-cell-right-color !important;
+}
+
+.bad_place {
+    background-color: $game-cell-almost-color !important;
+}
+
+#timer {
+    height: 2rem;
+}
+
+#keyboard button {
+    background-color: $navbar-background-color;
+    color: white;
+    font-size: 1.5rem;
+    width: 3rem;
+    height: 3rem;
+    border-style: solid;
+    border-width: 1px;
+    border-radius: 6px;
```

```suggestion
    border-radius: 6px;
    display:inline-flex;
    justify-content: center;
    align-items: center;
```

To correct the fact that the letter is not centered on mobile